### PR TITLE
Bonus GUI

### PR DESCRIPTION
**Pull request by PHIDIAS0303**
fix combined bonus
add auto personal battery recharge

### CODE REVIEW

File: modules/gui/bonus.lua [hunk 1]
```diff
@@ -146,6 +181,8 @@ Gui.element{
         end
     end
 
+    disp[disp['bonus_display_personal_battery_recharge'].tags.counter].caption = format_number(disp['bonus_display_personal_battery_recharge'].slider_value)
+
```
Comment (Cooldude2606) [inline modules/gui/bonus.lua:185]: The code you currently have causes and error, and when fixed it doesn't reset the value. So you need the code below.
```suggestion
    local slider = disp['bonus_display_personal_battery_recharge_slider']
    slider.slider_value = config.player_special_bonus['personal_battery_recharge'].value
    disp[slider.tags.counter].caption = format_number(slider.slider_value)

```

File: config/bonus.lua [hunk 2]
```diff
@@ -37,6 +36,18 @@ return {
         ['cpdb'] = 'character_item_pickup_distance_bonus'
         ]]
     },
+    player_special_bonus_rate = 300,
+    player_special_bonus = {
+        ['personal_battery_recharge'] = {
+            -- 1 MW
+            value = 8,
```
Comment (Cooldude2606) [inline config/bonus.lua:43]: This starting value is too high for the base number of points. A value of 2 would work, and although it is not centered, I do not see many people making use of it more than that.

**Review by Cooldude2606 — CHANGES_REQUESTED** [review]

File: config/bonus.lua (hunk 2 above)
Comment (PHIDIAS0303) [reply config/bonus.lua:43]: it is not that high when you are using it to do robots. 4 MW each

Comment (Cooldude2606) [reply config/bonus.lua:43]: Then please increase the base number of points, because the default exceeds it.

**Review by Cooldude2606 — APPROVED** [review]

**Pull request merged**